hack/tools.go

```diff
@@ -0,0 +1,6 @@
+package hack
+
+import (
+  "github.com/go-training/helloworld"
```

Comment:
From travis:

> hack/tools.go:4:3: import "github.com/go-training/helloworld" is a program, not an importable package

I think you need a `_` before the import (and a `gofmt`).

Reply:
Thanks :)

Re: whether we should keep this behavior; totally agree, ideally we would replicate the behavior of go build exactly, which might be a moving target.